package.xml

```diff
@@ -24,7 +24,6 @@
   <build_depend>tf2_ros</build_depend>
   <build_depend>laser_geometry</build_depend>
   <build_depend>angles</build_depend>
-  <build_depend>xmlrpcpp</build_depend>
```

Comment:
@bfjelds should eigen also be added as a build depend?

Reply:
I'm not sure :)

If I limit all references to eigen to laser_geometry, do I need a build_depend in laser_filters?

Reply:
If laser_filters doesn't directly use any eigen types or include any eigen headers, then no build depend should be needed on eigen in this package, you are right.

Reply:
laser_filters uses headers from laser_geometry that reference eigen … so in a way, laser_filters is doing the building that requires eigen (laser_geometry is just exporting the headers).  

that gets a little "it depends on what the definition of is is" to me :)

I figure that since you can't get to laser_geometry's headers without it exposing the build_depend on eigen, the build_depend isn't needed here.  That said, I'm a little dizzy from typing that.